test/messages/messages.e2e-spec.ts

```diff
@@ -121,4 +121,68 @@ describe("MessagesController (e2e)", () => {
       });
     });
   });
+
+  describe("GET classroom/:classroomId/messages", () => {
+    let teacherUser: User;
+    let teacherToken: string;
+    let classroom: Classroom;
+    let messageDto: CreateMessageDto;
+
+    beforeEach(async () => {
+      const {
+        user: teacherUserData,
+        teacher,
+        plainTextPassword: teacherPassword,
+      } = await UserFactory.createTeacher();
+      teacherUser = await createTeacherInDb(dbService, teacherUserData, teacher);
+      teacherToken = await getAccessToken(httpServer, teacherUser.email, teacherPassword);
+
+      classroom = await createClassroomInDb(dbService, teacherUser.teacher!);
+
+      messageDto = {
+        content: "Test message content",
+        classroomId: classroom.id,
+      };
+
+      await request(httpServer)
+        .post("/messages")
+        .set("Authorization", `Bearer ${teacherToken}`)
+        .send(messageDto)
+        .expect(HttpStatus.CREATED);
```

Comment:
Can't  you directly create in db with a helper rather than making a request? 
What if we had to make multiple messages? Wouldn't we then have to make multiple post requests to our mock server?

Reply:
Bhaiya, I have used this approach because I don't need to create messages for any other tests.